vendor/ezurio/rs1xx-ext-temp-1w-sensor.yaml

```diff
@@ -0,0 +1,187 @@
+name: Sentrius RS1xx Ext Temp 1W Sensor
+description: The Ezurio Sentrius™ RS1xx Temp/-RH has an external temperature sensor probe and humidity sensor making it a perfect fit for industrial applications. It uses LoRaWAN® and Bluetooth Low Energy (BLE) for data communication. The robust IP65 sensor enclosure allows it to be used in harsh environments.
+
+# Hardware versions (optional, use when you have revisions)
+hardwareVersions:
+  - version: 'rev 4'
+    numeric: 1
+
+# Firmware versions (at least one is mandatory)
+firmwareVersions:
+  - # Firmware version
+    version: '6.1_20_6_16'
+    numeric: 1
+    # Corresponding hardware versions (optional)
+    hardwareVersions:
+      - 'rev 4'
+
+    # LoRaWAN Device Profiles per region
+    # Supported regions are EU863-870, US902-928, AU915-928, AS923, CN779-787, EU433, CN470-510, KR920-923, IN865-867, RU864-870
+    profiles:
+      EU863-870:
+        # Unique identifier of the profile (lowercase, alphanumeric with dashes, max 36 characters)
+        id: rs1xx-sensor-profile
+        lorawanCertified: false
+        codec: rs1xx-ext-temp-1w-sensor-codec
+      US902-928:
+        id: rs1xx-sensor-profile
+        lorawanCertified: false
+        codec: rs1xx-ext-temp-1w-sensor-codec
+      AU915-928:
+        id: rs1xx-sensor-profile
+        lorawanCertified: false
+        codec: rs1xx-ext-temp-1w-sensor-codec
+      AS923:
+        id: rs1xx-sensor-profile
+        lorawanCertified: false
+        codec: rs1xx-ext-temp-1w-sensor-codec
+
+# Sensors that this device features (optional)
+# Valid values are: accelerometer, altitude, auxiliary, barometer, battery, button, co2, distance, dust, gps, gyroscope,
+# humidity, light, link, magnetometer, moisture, ph, pir, proximity, rssi, snr, sound, temperature, tvoc, velocity,
+# vibration, water, wind direction and wind speed.
+sensors:
+  - temperature
+
+# Dimensions in mm (optional)
+# Use width, height, length and/or diameter
+dimensions:
+  width: 91
+  length: 131
+  height: 34
+
+# Weight in grams (optional)
+weight: 185
+
+# Battery information (optional)
+battery:
+  replaceable: true
+  type: AA
+
+# Operating conditions (optional)
+operatingConditions:
+  # Temperature (Celsius)
+  temperature:
+    min: -25
+    max: 50
+  # Relative humidity (fraction of 1)
+  relativeHumidity:
+    min: 0
+    max: 1
+
+probeoperatingConditions:
+  # Temperature (Celsius)
+  temperature:
+    min: -55
+    max: 125
+
+# IP rating (optional)
+ipCode: IP65
+
+# Key provisioning (optional)
+# Valid values are: custom (user can configure keys), join server and manifest.
+keyProvisioning:
+  - custom
+  - join server
+
+# Key security (optional)
+# Valid values are: none, read protected and secure element.
+keySecurity: read protected
+
+# Product and data sheet URLs (optional)
+productURL: https://www.Ezurioconnect.com/wireless-modules/lorawan-solutions/sentrius-rs1xx-lora-enabled-sensors
```

Comment:
Domain name should be lowercase

Reply:
https://www.ezurio.com/iot-devices/lorawan-iot-devices/sentrius-rs1xx-lora-enabled-sensors